bin.src/imsim.py

```diff
@@ -32,6 +39,12 @@ def main():
     parser.add_argument('--log_level', type=str,
                         choices=['DEBUG', 'INFO', 'WARN', 'ERROR', 'CRITICAL'],
                         default='INFO', help='Logging level. Default: "INFO"')
+    parser.add_argument('--psf', type=str, default='DoubleGaussian',
+                        choices=['DoubleGaussian', 'Kolmogorov'],
```

Comment:
Do we want the default to be doubleGaussian or Kolmogorov?  I thought we wanted Kolmogorov...